docker/unauthorized-rce/README.md

```diff
@@ -1,23 +1,27 @@
-# docker daemon api 未授权访问漏洞
+# docker daemon api exploit
 
-参考链接：
+[中文版本(Chinese version)](README.zh-cn.md)
+
+Reference Links：
 
 - http://www.loner.fm/drops/#!/drops/1203.%E6%96%B0%E5%A7%BF%E5%8A%BF%E4%B9%8BDocker%20Remote%20API%E6%9C%AA%E6%8E%88%E6%9D%83%E8%AE%BF%E9%97%AE%E6%BC%8F%E6%B4%9E%E5%88%86%E6%9E%90%E5%92%8C%E5%88%A9%E7%94%A8
 
-## 漏洞环境
+## Vulnerability environment
 
-编译及启动漏洞环境：
+Enter following command to build and run the vulnerability environment:
 
 ```
 docker-compose build
 docker-compose up -d
 ```
 
-环境启动后，将监听2375端口。
+after the service started, the 2375 port was published to the host
+
+## Exploit
 
-## 漏洞复现
+Start a container, and mount the host `/etc` folder to the container, then we have read/write access to any files.
```

Comment:
Start a container, and mount the host's `/etc` folder to the container, then we will have read/write access to any file.